Documentation/running-fly-stage1.md

```diff
@@ -0,0 +1,77 @@
+# Running rkt with the *fly* stage1
+
+The *fly* stage1 is an alternative stage1 that runs a single-application ACI with only `chroot`-isolation.
+
+
+## How does it work?
+
+In comparison to the default stage1, there is no process manager involved in the stage1.
+
+The rkt application sets up the mounts for `/dev`, `/proc`, `/sys` and the user-provided volumes.
+Then, rkt enters the application's RootFS using the `chroot` syscall and finally executes the application.
+
+Here's a comparison of the default and fly stage1:
+
+stage1-coreos.aci:
+
+```
+host OS
+  └─ rkt
+    └─ systemd-nspawn
+      └─ systemd
+        └─ chroot
+          └─ user-app1
+```
+
+
+stage1-fly.aci:
+
+```
+host OS
+  └─ chroot
+    └─ user-app1
+```
+
+
+## Getting started
+
+You can either use `stage1-fly.aci` from the official release, or build rkt yourself with the right options:
+
+```
+$ ./autogen.sh && ./configure --with-stage1-flavors=fly && make
+```
+
+For more details about configure parameters, see [configure script parameters documentation](build-configure.md).
+This will build the rkt binary and the LKVM stage1-kvm.aci in `build-rkt-0.13.0+git/bin/`.
+
+### Selecting stage1 at runtime
+
+Curious readers can read a whole document on how to [choose which stage1.aci to use at runtime](https://github.com/coreos/rkt/blob/master/Documentation/commands.md#use-a-custom-stage-1).
+
+Here is a quick example of how to use a container stage1 named `stage1-fly.aci` in `/usr/local/rkt/`:
```

Comment:
Then we need to update the LKVM stage1 doc too. I suggest to do this afterwards and all together.